app/src/main/java/agency/five/codebase/android/movieapp/ui/component/CrewItem.kt

```diff
@@ -0,0 +1,40 @@
+package agency.five.codebase.android.movieapp.ui.component
+
+import agency.five.codebase.android.movieapp.mock.MoviesMock
+import androidx.compose.foundation.layout.Column
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.sp
+
+data class CrewItemViewState(
+    val name: String,
+    val job: String
+)
+
+@Composable
+fun CrewItem(
+    crewItemViewState: CrewItemViewState,
+) {
```

Comment:
Every component should expose `modifier: Modifier` as parameter. You can read more about it [here](https://chris.banes.dev/posts/always-provide-a-modifier/). 

Than use this `modifier` at the top-level composable function in components hierarchy (in this case, pass modifier to `Column` composable).